neps/nep-0463.md

```diff
@@ -0,0 +1,138 @@
+---
+  NEP: 463
+  Title: Decentralized Autonomous Organization Standard
+  Author: DAO Builders Group
+  Status: Draft
+  DiscussionsTo: nearbuilders.com/tg-dao
+  Type: Standards
+  Created: 21-Feb-2023
+---
+
+## Summary
+
+A standard interface to support decentralized autonomous organizations (DAOs) that enable proposals with governance.
+
+## Motivation
+
+There is a need for **interoperable DAO systems** on NEAR and other blockchains.
+
+Essentially, this NEP requests a review of the minimum viable Sputnik DAO.
```

Comment:
Let's remove sputnik dao, sputnik dao is a reference implementation that might adopt this standard but not necessarily what the standard should be.  Working backwards from use cases and applications in motivation how does the interface address this?